Scripts/Package.py

```diff
@@ -132,7 +132,6 @@ def mkdir_p(path):
 elif args.os == 'mac':
   if args.build == 'full' or args.build == 'core' or args.build == 'engine':
     src_exe = os.path.join(base_path, 'RetroFE', 'Build', 'retrofe')
-    shutil.copy(src_exe, output_path)
```

Comment:
reason for removal?

Reply:
it throws a non-existent directory error because output path ends with `RetroFE` when it should be `RetroFE.app` trying to place the executable into the mac app bundle
upon removal discovered 'shutil.copy(src_exe, output_path + '/RetroFE.app/Contents/MacOS/')' on line 138 places executable within the app bundle regardless and its omission does not affect expected functionalilty